src/App.js

```diff
@@ -0,0 +1,19 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import './App.css';
+import Header from './components/Header';
+import Books from './components/Books';
+import Categories from './components/Categories';
+
+function App() {
```

Comment:
- [ ] Great job so far 👏🏻 However I'm getting this error from the `/App.css';` file actually you're trying to import a file that does not exist in your project dir, please remove this line `import './App.css';` to fix the issue.

![image](https://github.com/Lawmsangi/bookstore-react/assets/79058364/ac89333a-fa39-408c-b687-0a2fbb8a331c)